src/main/java/Duke.java

```diff
@@ -1,50 +1,125 @@
 import java.util.Scanner;
 public class Duke {
+    private static final int LIST_SIZE = 100;
+    private static int taskNum = 0;
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-        System.out.println("Hello! I'm Duke");
-        System.out.println("What can I do for you?");
-
+        showGreetings();
         String command = "";
         Scanner in = new Scanner(System.in);
 
-        int taskNum = 0;                            // number of tasks stored
-        Task[] storedTasks = new Task[100];         // array of stored texts
+        Task[] storedTasks = new Task[LIST_SIZE];         // array of stored texts
         while (!command.equals("bye")) {
```

Comment:
Good handling of indentation. While loop has even more for loops inside but code segments are indented so it is clear which block belongs inside and outside the for loop.